composer.json

```diff
@@ -15,11 +15,18 @@
         }
     ],
     "require": {
-        "php": ">=5.4.0",
-        "aura/project-kernel": "~2.0",
-        "aura/cli": "~2.0",
+        "php": ">=7.2.0",
+        "aura/project-kernel": "4.x-dev",
```

Comment:
I think you should use dev-4.x . The github actions is failing due to composer issue.

Reply:
```diff
$ git diff
diff --git a/composer.json b/composer.json
index ecc1169..8635e0f 100644
--- a/composer.json
+++ b/composer.json
@@ -16,7 +16,7 @@
     ],
     "require": {
         "php": ">=7.2.0",
-        "aura/project-kernel": "4.x-dev",
+        "aura/project-kernel": "dev-4.x",
         "aura/cli": "update-for-php7-dev",
         "aura/dispatcher": "~2.0"
     },
```

```sh-session
$ composer update
Loading composer repositories with package information                                                         Updating dependencies                                 
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Root composer.json requires aura/project-kernel dev-4.x, found aura/project-kernel[dev-coverage, dev-master, dev-develop-2, 2.0.0-beta1, ..., 2.x-dev, 4.x-dev] but it does not match the constraint.
...
```

Reply:
@harikt `4.x-dev` is correct. The github action fail is because it is a branch.
We should use `^4.0`. We need 4.0.0 release.

Reply:
Great. I got a bit confused.